script/docker/server

```diff
@@ -12,4 +12,4 @@ cd "$(dirname "$0")/../.."
 # in the background and then use attach to create an interactive prompt.
 docker-compose up --detach
 
-docker attach scenic-or-not_web_1
```

Comment:
Although not ideal, this will need to be put back. The way we tend to get around this across docker compose projects is to tick "Use Docker Compose V2" in the Docker Desktop preferences.

Given the commit it's included in is tackling a different issue I suspect this was left in by accident as you needed to do this to get the app working?